google/cloud/aiplatform/experimental/vertex_model/utils/source_utils.py

```diff
@@ -20,6 +20,27 @@
 from typing import Dict
 from typing import Tuple
 
+import ast
+from collections import namedtuple
+
+Import = namedtuple("Import", ["module", "name", "alias"])
+
+
+def get_imports(path):
```

Comment:
It seems like we are parsing the imports and then composing them together again. It may be simpler to use the nodes directly to create the import lines:

Something like:

```Python
import ast

def get_import_lines(path):
    with open(path) as f:
        root = ast.parse(f.read(), path)

    for node in ast.iter_child_nodes(root):
        line = ''
        if isinstance(node, ast.ImportFrom):
            line+=f'from {node.module} '
            
        if isinstance(node, (ast.Import, ast.ImportFrom)):
            line+='import '
            for i, name in enumerate(node.names):
                line+=f'{name.name}'
                if name.asname:
                    line+= f' as {name.asname}'
                if len(node.names) > 0 and i < len(node.names) - 1:
                    line+=', '
        yield line
```

Reply:
got it, thank you!